packages/rest/test/unit/rest.server/rest.server.open-api-spec.unit.ts

```diff
@@ -116,6 +139,46 @@ describe('RestServer.getApiSpec()', () => {
     });
   });
 
+  it('ignores routes marked as "x-internal" via app.route(..., Controller, method)', () => {
+    class GreetController {
+      greet() {}
+    }
+
+    class MeetController {
+      meet() {}
+    }
+
+    server.route(
+      'get',
+      '/greet',
+      {'x-internal': true, responses: {}},
+      GreetController,
+      createControllerFactoryForClass(GreetController),
+      'greet',
+    );
+
+    server.route(
+      'get',
+      '/meet',
+      {responses: {}},
```

Comment:
Same comment as above - let's keep this test focused on the x-internal method only and don't duplicate the tests verifying how non-internal routes are handled.

Reply:
Same thing, verify no side-effects.

Reply:
Same thing as above.